cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,48 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
-
-  });
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import homePageObject from '../support/pages/home.pageObject';
+import accountPageObject from '../support/pages/account.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
 
-  beforeEach(() => {
-
-  });
-
-  it('should provide an ability to update username', () => {
+const settingsPage = newSettingsPageObject();
+const homePage = newhomePageObject();
+const accountPage = newaccountPageObject();
+const signInPage = newSignInPageObject();
 
+describe('Settings page', () => {
+  let user;
+    settingsPage.visit();
+    settingsPage.typeUsername(user2.username);
+    settingsPage.clickUpdateSettingsBtn();
+    homePage.assertHeaderContainUsername(user2.username);
   });
 
   it('should provide an ability to update bio', () => {
-
-  });
-
-  it('should provide an ability to update an email', () => {
+  describe('Settings page', () => {
+    signInPage.typeEmail(user2.email);
+    signInPage.typePassword(user.password);
+    signInPage.clickSignInBtn();
+    homePage.assertHeaderContainUsername(user2.email);
```

Comment:
The assertion is checking for the email instead of the username. It should assert the username to match the test scenario.